packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -23,6 +23,8 @@ const selectors = {
 	postToolbar: '.edit-post-header',
 	settingsToggle: '[aria-label="Settings"]',
 	saveDraftButton: '.editor-post-save-draft',
+	// there's a hidden button also with the "Preview" text, so using unique class name instead of text selector
+	previewButton: '.edit-post-header .editor-post-preview',
```

Comment:
This is perfectly fine, as I am finding on the proper mobile viewport (<480px) the button text is hidden.